### ISSUE TITLE
ugly call with long binary expressions

### ISSUE THREAD

**Issue by alexander-akait** [prettier/prettier#5172]
**Prettier 1.14.3**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAucBDANugFAHSgAkPwDMBXKMGAS2iwEp9g8iX8AnOGUtggRgG5mRAL4AaIYQAkAN1Rt8AfgX4ZcxctXylK2Vo271Otds2HTJg9oDk8AM4wr+PHX4hRICAAdq0W8lBybBAA7gAKcgh+KKjSEFQAJm4gAEZsqGAA1pwAyp7pVFAA5sgwbKRw7gW2cGwwoWmFALaoyMQY1e4AVrYAHgBCaZk5qI1wADIFcK3tFSB5bNVsyCmoyQCe6NBJnmwFMADqCTAAFsgAHAAM7jsQ1ftpnss7cIvSU+4cAI6kVBz1qE0Wkg2ugOiBqo0qCUyrNbAVCug4ABFUgQeDTUGzGCrQ7xE7IABM7lKqCo6HhAGEII1mssoNB3iBSNUACqrKIg6rCYRAA)
```sh
--parser babylon
```

**Input:**
```jsx
call(
    function() {
        return 1;
    },
    $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? 'test' 
);
```

**Output:**
```jsx
call(function() {
  return 1;
}, $var ??
  $var ??
  $var ??
  $var ??
  $var ??
  $var ??
  $var ??
  $var ??
  $var ??
  "test");

```

**Expected behavior:**

```jsx
call(
    function() {
        return 1;
    },
    $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? 'test' 
);
```

**Comment by sosukesuzuki** [comment]
This happen on SequenceExpression too.
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAKAOlABDrAzAVyjBgEtotUBKLYTXXAJzhgMewEZ7cBfAGm44AJADcAhoywB+KVlETps+ZJlzxKpesVqFq5dv16tqgOTwAzjBNZMVANwg+ICAAcy0c8lATGEAO4AChIInihiIhCkACaOIABGjGJgANYsAMouSaRQAObIMIwEcE4AFjAAtgA2AOolpBaZYHBpIfWkIvUAnsjg5p5O2eZwjDABiTnlYsh4YpVDTgBW5gAeAEKJKeli5XAAMtlw07PzIEvLadk5lXAAigQQ8EdzxSCZjEOMPXFicZ2V0LEXIxsjBqtEYCVkAAOAAMTiBECG1USLh6QLgHxEhycAEd7vAxq5QiAxOYALRQOBwKLU2LMPGkZhjMQTKZIGbPJxDcqkfKFF7mS7XO4PQ7s44vGA-MFRCHIABMTgKYlIlUuAGEIOVJj0oNBsSACEMACo-UIcoY8HhAA)
**Inpu:t**
```js
(
  function () {},
  $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? 'test' 
);
```

**Output:t**
```js
(function() {},
  $var ??
    $var ??
    $var ??
    $var ??
    $var ??
    $var ??
    $var ??
    $var ??
    $var ??
    "test");
```
**Expected:**
```js
(
  function () {},
  $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? $var ?? 'test' 
);
```

**Comment by thorn0** [comment]
Duplicate of https://github.com/prettier/prettier/issues/2456

**Comment by thorn0** [comment]
Not really a duplicate.

**Comment by thorn0** [comment]
It's a duplicate after all. Closing in favor of #2456
